components/Checkbox.js

```diff
@@ -41,6 +41,7 @@ const Checkbox = ({ className, id, labelText, onChange, ...other }) => {
           <Icon
             className="bx--checkbox-checkmark"
             description="checkmark"
+            alt="checkmark"
```

Comment:
for these `alt` tags and below, could you actually move them to a `defaultProp` and then reference that prop here? It'll allow for people to pass in different internationalized strings if need be